docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -354,7 +354,7 @@ class ComponentExample extends React.PureComponent<IComponentExampleProps, IComp
     }
 
     return (
-      <Provider theme={mergeThemes(themes.teams, newTheme)}>
+      <Provider theme={newTheme}>
```

Comment:
Good find!  I was wondering what this was from 😊 There is now a dead import to `mergeThemes` in this file that needs to be cleaned up.

Reply:
Resolved in #228